packages/react-ui/components/Button/Button.tsx

```diff
@@ -420,7 +421,11 @@ export class Button extends React.Component<ButtonProps, ButtonState> {
     let loadingNode = null;
     if (loading && !icon) {
       const loadingIcon = _isTheme2022 ? <LoadingIcon size={size} /> : <Spinner caption={null} dimmed type="mini" />;
-      loadingNode = <div className={styles.loading()}>{loadingIcon}</div>;
+      loadingNode = (
+        <div data-tid={_isTheme2022 ? ButtonDataTids.spinner : undefined} className={styles.loading()}>
```

Comment:
Думаю, новый дата-тид без условий можно ставить без условий.
Использовать его в любом случае правильнее, чем дата-тид от Спинера. А наличие сразу обоих ничего не ломает.

При возможности стоит не добавлять лишний раз условие `isTheme2022`. При выпиливании старой темы будет меньше работы.
```suggestion
        <div data-tid={ButtonDataTids.spinner} className={styles.loading()}>
```

Reply:
Удалил условие